SemVerAnalyzer.sln

```diff
@@ -0,0 +1,63 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I wonder if these files always start with a blank line by way of detecting which linebreaks are present...